utils/TimeUtils.ts

```diff
@@ -0,0 +1,46 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { STORAGE_KEYS } from "./AsyncStoageUtils";
+
+// the length of a day in milliseconds
+const DAY_IN_MS = 24 * 60 * 60 * 1000;
+const EXACT_DATE = new Date();
+// standardized current date that does not factor in the time it was created, just the date
+export const getCurrentDate = () =>
+  new Date(
+    EXACT_DATE.getFullYear(),
+    EXACT_DATE.getMonth(),
+    EXACT_DATE.getDate(),
+    0
+  ).toISOString();
+
+export const calculateRecordingDay = (startDay: string) => {
+  // check which "day" it is for them (day 1, 2, 3) based on the start day.
+  // When you subtract them, it should only ever be 0 (same day), DAY_IN_MS (1 day later) or DAY_IN_MS * 2 (2 days later) since
+  // current date and start day should always be the same time of day
```

Comment:
Is it safe to assume that the user will always record at the same time everyday?

Reply:
Maybe? I believe it does not matter, but who knows, you can never trust date math (https://yourcalendricalfallacyis.com)